doc/tutorial/code/double.saw

```diff
@@ -3,7 +3,7 @@ double_imp <- llvm_extract l "double_imp";
 double_ref <- llvm_extract l "double_ref";
 let thm = {{ \x -> double_ref x == double_imp x }};
 
-r <- prove yices thm;
+r <- prove abc thm;
```

Comment:
If I understand 02092f388bfa3431e277f41beeb55893e78d63eb, which introduced this change, this was deliberately changed to `yices` for performance reasons. It might be better to just delete the redundant `prove yices thm` statements below and update the tutorial accordingly.

Reply:
Thanks. Now that you've changed the source code for the tutorial, can you also regenerate the `doc/tutorial/sawScriptTutorial.pdf` file that uses this code? You should be able to do so by installing `pandoc`, navigating to the `doc/tutorial` directory, and then running `make`.